Content.Shared/Climbing/SharedClimbingSystem.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.Movement;
+using Robust.Shared.GameObjects;
+
+namespace Content.Shared.Climbing
+{
+    public abstract class SharedClimbingSystem : EntitySystem
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<SharedClimbingComponent, MovementAttemptEvent>(HandleMove);
+        }
+
+        private void HandleMove(EntityUid uid, SharedClimbingComponent component, MovementAttemptEvent args)
+        {
+            if (!component.GetIsTransitioning()) return;
+            args.Cancel();
+        }
```

Comment:
Same as above. Also climbing checking IsTransitioning will be removed someday so not much point having this tbh.

Reply:
Actually just slap a TODO on this and we'll deal with it later.